enkit/version/version.go

```diff
@@ -0,0 +1,42 @@
+package version
+
+import (
+	"fmt"
+
```

Comment:
is the extra line on purpose?

Reply:
Yes, to separate stdlib imports from our imports